libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/model/eh/EHorizonMapper.kt

```diff
@@ -63,11 +63,11 @@ internal suspend fun RoadObjectDistanceInfo.mapToEHorizonObjectDistanceInfo():
 /**
  * Map the ElectronicHorizonPosition.
  */
-internal fun RoadObjectLocation.mapToEHorizonObjectLocation(): EHorizonObjectLocation {
+internal fun GraphPath.mapToEHorizonObjectLocation(): EHorizonObjectLocation {
     return EHorizonObjectLocation(
-        edges,
-        percentAlongBegin,
-        percentAlongEnd
+        this.edges,
+        this.percentAlongBegin,
+        this.percentAlongEnd
     )
 }
```

Comment:
cc @korshaknn, I just tried to recreate the old behavior

Reply:
will fix in my PR

Reply:
Cherry-picked the fixes.